force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -101,19 +158,36 @@ public with sharing class ClassesObjectsMethods {
      * @return The number of vowels in the string.
      */
     public static Integer countVowels(String str) {
-        // Initialize the result Integer
-
-        /*
-        for () {
+        //Initialize the result Integer
+        Integer count = 0;
+        List<String> vowels = new List<String>{'a','e','i','o','u'};
+        // // Loop through the string
+        // // Compare the character to the vowels set
+        // // If the character is a vowel, increment the result integer
+        // // return the result Integer
+       
+        for (Integer i = 0; i < str.length(); i++)
+        {  
+            String eachLetter = str.substring(i, i + 1);
+            if (vowels.contains(eachLetter))
+            {
+                count++;
+            }
+                
+        }       
             // get individual characters from the string - hint use substring or split
             // check if the character is a vowel
             // if it is a vowel, increment the result Integer
```

Comment:
Be sure to remove duplicate (or wandering) comments from your code prior to submission.

Reply:
It seems that I have a habit of leaving them there. I'll work on cleaning my comments up.